content/how-to/ignore-rules/_index.md

```diff
@@ -1,7 +1,7 @@
 ---
-title: "Configure ignore rules"
-description: "How to set ignore rules"
-lead: ""
+title: "How to set ignore rules"
```

Comment:
"how to" is in the URL and implied in by the "user guide" section. I don't think we should add it to all these titles.